sphinx/source/start_network.rst

```diff
@@ -89,7 +89,7 @@ Once done, each additional node (here, node 1) can join the existing network by
 
     $ client --server-address=node1_ip:node1_rpcport --ca=node1_cert_file joinnetwork --req=@joinNetwork.json
 
-When executing the ``joinNetwork.json`` RPC, the target node initiates an enclave-to-enclave TLS connection to the network leader to retrieve the network secrets required to decrypt the serialised replicated transactions. Once the join protocol completes, the new node becomes a follower of the Raft network and starts replicating transactions executed by the leader.
+When executing the ``joinNetwork.json`` RPC, the target node initiates an enclave-to-enclave TLS connection to the network primary to retrieve the network secrets required to decrypt the serialised replicated transactions. Once the join protocol completes, the new node becomes a backup of the consensus network and starts replicating transactions executed by the primary (current consensus protocol implemented is Raft and thus backup corresponds to follower).
```

Comment:
I don't think we should repeat this statement multiple times. In this context, I think we can talk entirely about some abstract Consensus model and not mention Raft at all. We can define primary and backup in `glossary.rst`